src/components/study-pane.jsx

```diff
@@ -68,28 +69,29 @@ const StudyPane = ({ studyUuid, currentNode, currentRootNetworkUuid, ...props })
     }, []);
 
     return (
-        <>
-            {/*Rendering the map is slow, do it once and keep it display:none*/}
-            <div
-                className="singlestretch-child"
-                style={{
-                    display: props.view === StudyView.MAP ? null : 'none',
-                }}
-            >
-                <MapViewer
-                    studyUuid={studyUuid}
-                    currentNode={currentNode}
-                    currentRootNetworkUuid={currentRootNetworkUuid}
-                    view={props.view}
-                    openDiagramView={openDiagramView}
-                    tableEquipment={tableEquipment}
-                    onTableEquipementChanged={(newTableEquipment) => setTableEquipment(newTableEquipment)}
-                    onChangeTab={props.onChangeTab}
-                ></MapViewer>
-            </div>
-            {/* using a key in these TabPanelLazy because we can change the nodeUuid in this component */}
-            <TabPanelLazy key={`spreadsheet-${currentNode?.id}`} selected={props.view === StudyView.SPREADSHEET}>
-                <Paper sx={styles.table}>
+        <Box sx={styles.table}>
```

Comment:
`styles.table` was a style for the Spreadsheet tab only, if you move it here, then rename it, because it's not a table anymore...

Reply:
OK, I will rename containerBox

Reply:
renamed as paneContainer and add also tabsContainer for the box which wrap tabs